docker-compose.yml

```diff
@@ -78,3 +78,9 @@ services:
       - ./data/files:/share
     restart: always
     logging: *default-logging
+  publicationTasksGenerator:
```

Comment:
Kebab case gebruiken ipv camel case (zoals je in de config/delta/rules.js wel gedaan hebt)